companies/4sigma.json

```diff
@@ -8,7 +8,7 @@
         "telecommunication"
     ],
     "name": "4sigma GmbH",
-    "address": "Bajuwarenring 9\n82041 Oberhaching\nDeutschland",
+    "address": "Bajuwarenring 9          \n   \t82041 Oberhaching   \n   Deutschland",
```

Comment:
🚫 **[recordlint]** <[trim-address-lines](https://github.com/datenanfragen/data#addresses)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>The lines of the `address` aren't trimmed (at least one line contains trailing or leading whitespace).
```suggestion
    "address": "Bajuwarenring 9\n82041 Oberhaching\nDeutschland",
```